### ISSUE TITLE
Load Configuration File via HTTPS

### ISSUE THREAD

**Issue by gianlucafrei** [gianlucafrei/nellygateway#25]
Currently we Nelly only supports to load a config file from the local filesystem. We should add the option to load a configuration file via HTTPS.

-> Add a new class that implements NellyConfigLoader that load the file via https from remote
-> Implement the HTTP loading if the file path begins with https://
-> Add error handling

**Comment by nbaars** [comment]
Maybe integrate with Spring Cloud Config server? (https://cloud.spring.io/spring-cloud-config/reference/html/)

**Comment by gianlucafrei** [comment]
@nbaars Seems like a good idea, however, only HTTPS is way simpler. Can Spring Cloud Config host our custom YAML file?

**Comment by nbaars** [comment]
Yes, by default it will use a Git repo it will checkout but you can also define a different backend, for example: Vault, file, JDBC etc etc, see: https://cloud.spring.io/spring-cloud-config/multi/multi__spring_cloud_config_server.html#_file_system_backend
